.stylelintrc

```diff
@@ -2,12 +2,7 @@
   "extends": "stylelint-config-sass-guidelines",
   "rules": {
     "max-nesting-depth": 2,
-    "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
```

Comment:
I'd prefer to keep these rules as is and instead disable them inline where the custom site styles are.

If you add the following before the `.tui-html-code` selector in `terra-ui.com/app/modules/terra-ui.scss`, it should lint without erroring out on the custom site styles selector pattern.

```
// Custom site styles not follow terra selector-class-pattern
// stylelint-disable selector-class-pattern
// background/padding on code
.tui-html-code {
  background-color: #f7f7f9;
  padding: 1rem;
}
```